charts/platform/values.yaml

```diff
@@ -112,17 +131,12 @@ otelCollector:
   database: "default"
 
   resources:
-    {}
-    # We usually recommend not to specify default resources and to leave this as a conscious
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    # limits:
-    #   cpu: 1
-    #   memory: 2Gi
-    # requests:
-    #   cpu: 200m
-    #   memory: 400Mi
+    limits:
+      cpu: 0.1
+      memory: 256Mi
+    requests:
+      cpu: 0.1
+      memory: 256Mi
```

Comment:
Is this enough for the Otel collector? 🤔

Reply:
I'll probably set it as:

```
resources:
  limits:
    cpu: 500m
    memory: 512Mi
  requests:
    cpu: 1000m
    memory: 1Gi
```

To handle bursts. Until we use it we won't really know any better.